src/V3ThreadPool.cpp

```diff
@@ -53,6 +57,43 @@ void V3ThreadPool::resize(unsigned n) VL_MT_UNSAFE VL_EXCLUDES(m_mutex)
     }
 }
 
+void V3ThreadPool::suspendMultithreading() VL_MT_SAFE VL_EXCLUDES(m_mutex)
+    VL_EXCLUDES(m_stoppedJobsMutex) {
+    V3LockGuard stoppedJobsLock{m_stoppedJobsMutex};
+    if (!m_workers.empty()) { stopOtherThreads(); }
+
+    if (!m_mutex.try_lock()) {
+        v3fatal("Tried to suspend thread pool when other thread uses it.");
+        assert(0);  // LCOV_EXCL_LINE
+        VL_UNREACHABLE;
```

Comment:
```suggestion
```

v3fatal should handle doing this.  If somehow lock analysis doesn't see it properly, first try UASSERT, or fix v3fatal to do it.

Reply:
It turned out that ``v3fatal`` requires ``VL_ATTR_NORETURN`` for clang to correctly detect this function doesn't return.